tests/printer/printer.cpp

```diff
@@ -768,3 +768,213 @@ TEST(Printer, noChangeToManualIds)
     auto printer = libcellml::Printer::create();
     EXPECT_EQ(in, printer->printModel(model, true));
 }
+
+const std::string mathHeader = "<math xmlns=\"http://www.w3.org/1998/Math/MathML\" xmlns:cellml=\"http://www.cellml.org/cellml/2.0#\">\n";
+const std::string mathFooter = "</math>\n";
+const std::string prettyModelString =
```

Comment:
These should be at the top of the file and be declared in capitals `MATH_HEADER`.

Reply:
364144e8